### PR TITLE
fix: missing translations

### PR DESCRIPTION
**Pull request by Soare-Robert-Daniel**
### Summary

Added missing strings.

New pot file: [feedzy-rss-feeds.pot.zip](https://github.com/user-attachments/files/17522678/feedzy-rss-feeds.pot.zip)

### Will affect visual aspect of the product

NO

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes https://github.com/Codeinwp/feedzy-rss-feeds/issues/978

### CODE REVIEW

File: includes/layouts/feedzy-pro.php [hunk 1]
```diff
@@ -113,8 +119,18 @@ class="pro-label">Developer & Agency plans</span></h3>
             </ul>
             <ul class="fz-pro-features-table-row">
                 <li class="features-info">
-                    <h3 class="h3">OpenAI (ChatGPT) Support <span class="pro-label">Developer & Agency plans</span></h3>
-                    <p>Using the OpenAI API integration, paraphrase, summarize, generate missing featured images, or apply your custom prompt to the imported content.
+                    <h3 class="h3">
+                        <?php
+                        // translators: %s: OpenAI (ChatGPT)
+                        echo sprintf( esc_html__( '%s Support', 'feedzy-rss-feeds' ), 'OpenAI (ChatGPT)' );
```
Comment (HardeepAsrani) [inline includes/layouts/feedzy-pro.php]: Brand names are also translatable to localized characters.

File: includes/layouts/feedzy-pro.php [hunk 2]
```diff
@@ -113,8 +119,18 @@ class="pro-label">Developer & Agency plans</span></h3>
...
+                        ?>
+                        <span class="pro-label"><?php esc_html_e( 'Answer a few questions to help us improve Feedzy', 'feedzy-rss-feeds' ); ?></span>
+                    </h3>
+                    <p>
+                        <?php
+                        /* translators: %s: OpenAI API */
+                        echo sprintf( esc_html__( 'Using the %s integration, paraphrase, summarize, generate missing featured images, or apply your custom prompt to the imported content.', 'feedzy-rss-feeds' ), 'OpenAI API' );
```
Comment (HardeepAsrani) [inline includes/layouts/feedzy-pro.php]: Same here with brand name.

File: includes/layouts/feedzy-pro.php [hunk 3]
```diff
@@ -126,9 +142,20 @@ class="pro-label">Developer & Agency plans</span></h3>
...
             <ul class="fz-pro-features-table-row">
                 <li class="features-info">
-                    <h3 class="h3">WordAI & SpinnerChief integration <span class="pro-label">Agency plan</span></h3>
-                    <p>WordAi and SpinnerChief can be used to rephrase RSS feeds when they are imported as posts in
-                        WordPress.
+                    <h3 class="h3">
+                        <?php
+                        
+                        /* translators: %s: WordAI & SpinnerChief */
+                        echo sprintf( esc_html__( '%s integration', 'feedzy-rss-feeds' ), 'WordAI & SpinnerChief' );
```
Comment (HardeepAsrani) [inline includes/layouts/feedzy-pro.php]: Same here with brand name.

File: includes/layouts/feedzy-pro.php [hunk 4]
```diff
@@ -126,9 +142,20 @@ class="pro-label">Developer & Agency plans</span></h3>
...
+                        ?>
+                        <span class="pro-label"><?php esc_html_e( 'Agency plan', 'feedzy-rss-feeds' );?></span>
+                    </h3>
+                    <p>
+                        <?php
+                       
+                        /* translators: %1$s: WordAi, %2$s: SpinnerChief */
+                        echo sprintf( esc_html__( '%1$s and %2$s can be used to rephrase RSS feeds when they are imported as posts in WordPress.', 'feedzy-rss-feeds' ), 'WordAi', 'SpinnerChief' );
```
Comment (HardeepAsrani) [inline includes/layouts/feedzy-pro.php]: Same here with brand name.

File: js/ActionPopup/SortableItem.js [hunk 5]
```diff
@@ -168,7 +168,14 @@ const SortableItem = ({ propRef, loopIndex, item }) => {
 							<BaseControl>
 								<TextareaControl
 									label={ __( 'Main Prompt', 'feedzy-rss-feeds' ) }
-									help={__( 'You can use {content} in the textarea such as: "Rephrase my {content} for better SEO.".', 'feedzy-rss-feeds' )}
+									help={
+										sprintf(
```
Comment (HardeepAsrani) [inline js/ActionPopup/SortableItem.js:172]: We can use the same placeholder twice rather than adding it twice as an arg.

**Review by HardeepAsrani — APPROVED** [review]
Looks good, left a couple of NITs.

**Comment by Soare-Robert-Daniel** [comment]
As discussed, the brands will be separate translate strings so we can get reusability on general strings like `%s Support`

**Pull request merged**